test_pull_requests

```diff
@@ -124,23 +124,20 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
-# TODO: create a :state to "content" mapping for composing new bot
-# messages
-PR_STATES = {
-  /^Evaluating/ => :evaluating,
-  /^Running/ => :running,
-  /^Waiting/ => :waiting,
-  /^SUCCESS/ => :success,
-  /^FAILURE/ => :failure,
-  /^ABORTED/ => :aborted,
-  /^UNSTABLE/ => :unstable,
-  /^NOT_BUILT/ => :not_built,
-  /^#{ACTION_NOT_MERGE}/ => :action_not_merge,
-  /^#{ACTION_NOT_TEAM}/ => :action_not_team,
-  /^#{ACTION_UNSUPPORTED_BRANCH}/ => :action_unsupported_branch,
+CONTENT_TO_STATE = {
+  "Evaluating for testing" => :evaluating,
+  "Running" => :running,
+  "Waiting" => :waiting, #damn and blast
+  "SUCCESS" => :success,
+  "FAILURE" => :failure,
+  "ABORTED" => :aborted,
+  "UNSTABLE" => :unstable,
+  "NOT_BUILT" => :not_built
```

Comment:
It's one of the possible job statuses Jenkins can return. It's not used in t-p-r today, but I thought we could go ahead and store it here in case we ever want to handle it.